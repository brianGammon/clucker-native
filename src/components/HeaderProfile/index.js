import HeaderProfileComponent from './HeaderProfile';

export default HeaderProfileComponent;
