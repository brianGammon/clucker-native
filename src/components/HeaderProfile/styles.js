import { StyleSheet } from 'react-native';

export default StyleSheet.create({
  icon: {
    paddingLeft: 10,
    fontSize: 30,
    color: 'grey',
  },
});
