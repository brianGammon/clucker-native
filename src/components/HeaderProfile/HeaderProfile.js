import * as React from 'react';
import Ionicons from 'react-native-vector-icons/Ionicons';
import styles from './styles';

type Props = {
  navigation: any,
};

export default ({ navigation }: { navigation: Props.navigation }) => (
  <Ionicons onPress={() => navigation.toggleDrawer()} name="ios-person" style={styles.icon} />
);
