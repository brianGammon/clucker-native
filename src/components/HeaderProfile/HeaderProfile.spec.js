import React from 'react';
import HeaderProfile from '.';

describe('HeaderProfile component:', () => {
  test('Should render', () => {
    const wrapper = shallow(<HeaderProfile />);
    expect(wrapper).toMatchSnapshot();
  });
});
