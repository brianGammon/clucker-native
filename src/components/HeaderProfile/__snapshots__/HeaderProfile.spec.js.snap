// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderProfile component: Should render 1`] = `
<Icon
  allowFontScaling={false}
  name="ios-person"
  onPress={[Function]}
  size={12}
  style={
    Object {
      "color": "grey",
      "fontSize": 30,
      "paddingLeft": 10,
    }
  }
/>
`;
